packages/react/src/component-tests/IcCard/IcCard.cy.tsx

```diff
@@ -0,0 +1,770 @@
+    cy.checkA11yWithWait();
+  });
+
+  it("should render clickable button", () => {
+    mount(
+      <div style={{ margin: "16px" }}>
+        <IcCard
+          heading="Americano order"
+          subheading="Name: Michael"
+          message="Extras: Double espresso shot and oat milk."
+          clickable
+          onClick={(ev) => console.log(ev)}
+        >
+          <SlottedSVG
+            slot="icon"
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewBox="0 0 24 24"
+            width="24px"
+            fill="#000000"
+          >
+            <path d="M2,21V19H20V21H2M20,8V5H18V8H20M20,3A2,2 0 0,1 22,5V8A2,2 0 0,1 20,10H18V13A4,4 0 0,1 14,17H8A4,4 0 0,1 4,13V3H20M16,5H6V13A2,2 0 0,0 8,15H14A2,2 0 0,0 16,13V5Z" />
+          </SlottedSVG>
+        </IcCard>
+      </div>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("ic-card").click();
+    cy.compareSnapshot("clickableButtonCard", DEFAULT_TEST_THRESHOLD + 0.05);
+    cy.checkA11yWithWait();
+  });
+
+  it("should render with badge", () => {
+    mount(
+      <div style={{ margin: "16px" }}>
+        <IcCard
+          heading="Americano order"
+          subheading="Name: Michael"
+          message="Extras: Double espresso shot and oat milk."
+          clickable
+          href="#"
+        >
+          <IcBadge size="large" text-label="New" slot="badge" variant="info" />
+          <SlottedSVG
+            slot="icon"
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewBox="0 0 24 24"
+            width="24px"
+            fill="#000000"
+          >
+            <path d="M2,21V19H20V21H2M20,8V5H18V8H20M20,3A2,2 0 0,1 22,5V8A2,2 0 0,1 20,10H18V13A4,4 0 0,1 14,17H8A4,4 0 0,1 4,13V3H20M16,5H6V13A2,2 0 0,0 8,15H14A2,2 0 0,0 16,13V5Z" />
+          </SlottedSVG>
+        </IcCard>
+      </div>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("ic-badge").should(BE_VISIBLE);
+    cy.compareSnapshot("badgeCard", DEFAULT_TEST_THRESHOLD + 0.05);
+    cy.checkA11yWithWait();
+  });
+
+  it("render disabled", () => {
+    mount(
+      <IcCard
+        heading="Americano order"
+        subheading="Name: Michael"
+        message="Extras: Double espresso shot and oat milk."
+        clickable
+        disabled
+      >
+        <SlottedSVG
+          slot="icon"
+          xmlns="http://www.w3.org/2000/svg"
+          height="24px"
+          viewBox="0 0 24 24"
+          width="24px"
+          fill="#000000"
+        >
+          <path d="M2,21V19H20V21H2M20,8V5H18V8H20M20,3A2,2 0 0,1 22,5V8A2,2 0 0,1 20,10H18V13A4,4 0 0,1 14,17H8A4,4 0 0,1 4,13V3H20M16,5H6V13A2,2 0 0,0 8,15H14A2,2 0 0,0 16,13V5Z" />
+        </SlottedSVG>
+      </IcCard>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get('[disabled="true"]').should("exist");
+    cy.compareSnapshot("disabledCard", DEFAULT_TEST_THRESHOLD + 0.03);
+    cy.checkA11yWithWait();
+  });
+
+  it("render full width variant ", () => {
```

Comment:
As i've commented on the screenshot, the visual test here doesn't look any different from a normal card. Could you wrap the test in a <div> that has a specific width set to it, to test that the card does change width to fill its parent element?